### PR TITLE
CIV-10916 Claimant Intent – Check your Answers: How do you wish to pr…

### PR DESCRIPTION
**Pull request by ElifDenizEsea**
…oceed

**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-10916

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by jarekPierchala — APPROVED** [review]

File: src/main/services/features/claimantResponse/responseSection/buildYourResponseSection.ts [hunk 1]
```diff
@@ -36,7 +39,18 @@ export const buildPartAdmitPayImmediatelySummaryRows = (claim: Claim, claimId: s
 
 };
 
-export const getDoYouAgreeDefendantPaid = ( claim : Claim, claimId: string, lng: string): SummaryRow => {
+export const buildHowDoYourWantToProceed = (claim: Claim, claimId: string, lang: string): SummaryRow => {
+  const partAdmitAcceptedHref = constructResponseUrlWithIdParams(claimId, CLAIMANT_RESPONSE_SETTLE_ADMITTED_CLAIM_URL);
+  const selectedOption = claim?.claimantResponse?.chooseHowToProceed?.option;
```
Comment (laarmada) [inline src/main/services/features/claimantResponse/responseSection/buildYourResponseSection.ts:44]: you can remove the ? here in claim? always we will return a claim

**Review by laarmada — DISMISSED** [review]
minor comment

**Review by laarmada — APPROVED** [review]

**Review by portizdev — APPROVED** [review]

File: src/main/services/features/claimantResponse/checkAnswers/checkAnswersService.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ const logger = Logger.getLogger('claimantResponseCheckAnswersService');
 
 const buildSummarySections = (claimId: string, claim: Claim, lang: string): SummarySections => {
   const getYourResponseSection = () => {
-    return claim.isFullDefence() || claim.isPartialAdmission()
+    return claim.isFullDefence() || claim.isPartialAdmission() || claim.isFullAdmission()
```
Comment (laarmada) [inline src/main/services/features/claimantResponse/checkAnswers/checkAnswersService.ts:17]: which more options we have? ..... you are covering the 3 existing options right?

**Review by laarmada — CHANGES_REQUESTED** [review]

File: src/main/services/features/claimantResponse/checkAnswers/checkAnswersService.ts (hunk 2 above)
Comment (ElifDenizEsea) [reply src/main/services/features/claimantResponse/checkAnswers/checkAnswersService.ts:18]: I added parts for full admission. and it is covering

**Review by laarmada — DISMISSED** [review]

**Review by laarmada — DISMISSED** [review]

**Review by jeswanth-hmcts — DISMISSED** [review]

**Pull request closed without merging**